### ISSUE TITLE
rules in a category CAT in categories.java not enabled by --enable-categories CAT

### ISSUE THREAD

**Issue by Konfekt** [languagetool-org/languagetool#638]
With the snapshot from December 14th, the rule `ACCENTUATION_CHECK_PT` in Categories.java  in the category CONFUSED_WORD is not enabled by `--enable-categories CONFUSED_WORDS` To see this, write

No espirito vimos que isto não vale!

when there is no error message. Instead `-e ACCENTUATION_CHECK_PT`gives

ACCENTUATION_CHECK_PT Se é um nome ou um adjectivo, tem acento.

**Comment by danielnaber** [comment]
I guess the issue is that it's not (or not just) the category that's off by default, but also this specific rule (`ACCENTUATION_CHECK_PT`).

**Comment by TiagoSantos81** [comment]
You are right. I agreed to add this rule with the condition that it would be set to DefaultOff. See #610.  Dec 13 2016 comment.
I left this open, because this is the type of non issues that should not get attention. 

I suggest leaving it open, for reference purposes.
